test/Identity.js

```diff
@@ -661,6 +746,40 @@ describe('Identity model', function() {
         return w.getName() == 'treintaydos';
       }).should.deep.equal(w);
 
+      iden.addWallet(w2);
+      iden.getWalletById('33').getName().should.equal('treintaytres');
+      iden.walletIds.should.deep.equal(['32', '33']);
+
+    });
+
+    it('should read and bind wallet', function(done) {
+      iden.addWallet(w);
+      iden.storage.getItem = sinon.stub().yields(w, JSON.stringify(iden));
+      iden.readAndBindWallet('32', function(err) {
+        iden.getWalletById('32').getName().should.equal('treintaydos');
+        done();
+      });
+    });
+
+
+    it('should open wallet', function() {
+      iden.addWallet(w);
+      iden.storage.getItem = sinon.stub().yields(w, JSON.stringify(iden));
+      iden.readAndBindWallet = sinon.spy();
+      iden.openWallets();
+      iden.readAndBindWallet.should.calledOnce;
+
+    });
+
+    it('should open wallets', function() {
+      iden.addWallet(w);
+      iden.addWallet(w2);
+      iden.storage.getItem = sinon.stub().yields(w, JSON.stringify(iden));
+      iden.readAndBindWallet = sinon.spy();
+      iden.openWallets();
+      iden.walletIds.should.deep.equal(['32', '33']);
```

Comment:
I don't think this "proves" wallets were open. This array is filled by calling the #addWallet method